.github/workflows/automatic-update.yml

```
@@ -43,3 +44,9 @@ jobs:
 
             **Notes for reviewer:**
             This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
+
+      - name: Auto approve PR
+        uses: hmarr/auto-approve-action@v3
```

```suggestion
        uses: hmarr/auto-approve-action@v4
```

The [action readme](https://github.com/hmarr/auto-approve-action/tree/v4) suggests using v4 due to deprecated versions of node.